CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+Contributing to Mypy
+====================
+
+Welcome!  Mypy is a community project that aims to work for a wide
+swath of Python users and Python codebases.  If you're trying Mypy on
+your Python code, your experience and what you can contribute is
+important to the project's success.
+
+
+Getting started, building, and testing
+--------------------------------------
+
+If you haven't already, take a look at the project's README file and
+the [Mypy documentation](http://mypy.readthedocs.org/en/latest/), and
+try adding type annotations to your file and type-checking it with Mypy.
```

Comment:
I chatted with Jukka about this in person last week -- I think always lower-casing the term can cause it to get swallowed somewhat in a sentence, and as a proper name referring to the project and the body of software it makes sense to capitalize it in the same fashion as other proper names. When it's referring specifically to the command-line program `mypy`, when it's properly enclosed in backticks, it should be lowercase, but for the broader term I'd rather capitalize it. Similar to Git which is a version-control system and `git` which is a command, or Emacs which is a text editor and `emacs` which is a command.

Jukka was open to this convention but wanted to make a crisp decision on it. I'll open a separate thread.